Library/Homebrew/formula.rb

```diff
@@ -1526,8 +1527,10 @@ def undeclared_runtime_dependencies
     keg = opt_or_installed_prefix_keg
     return [] unless keg
 
-    linkage_checker = LinkageChecker.new(keg, self)
-    linkage_checker.undeclared_deps.map { |n| Dependency.new(n) }
+    DatabaseCache.new(:linkage) do |database_cache|
+      linkage_checker = LinkageChecker.new(keg, database_cache, false, self)
+      break linkage_checker.undeclared_deps.map { |n| Dependency.new(n) }
```

Comment:
Why is a `break` needed here?

Reply:
I wanted to return the dependencies `linkage_checker.undeclared_deps.map { |n| Dependency.new(n) }` in the function. If I remove the `break`, it causes a test to fail:

<img width="728" alt="screen shot 2018-05-06 at 3 58 11 pm" src="https://user-images.githubusercontent.com/8942499/39677288-661ea676-5146-11e8-8001-fedd2dc2c210.png">